src/BPEssentials/Commands/Moderation/StaffChat.cs

```diff
@@ -0,0 +1,30 @@
+﻿using BPEssentials.Abstractions;
+using BPEssentials.ExtensionMethods;
+using BPEssentials.Utils;
+using BrokeProtocol.Entities;
+
+namespace BPEssentials.Commands
+{
+    public class ToggleStaffChat : Command
+    {
+        public void Invoke(ShPlayer player, string text = "")
+        {
+            var ePlayer = player.GetExtendedPlayer();
+            if (string.IsNullOrEmpty(text))
```

Comment:
Avoid using `IsNullOrEmpty`, use `IsNullOrWhiteSpace` instead.

```suggestion
            if (string.IsNullOrWhiteSpace(text))
```